src/main/java/dev/ebullient/convert/tools/pf2e/Json2QuteCreature.java

```diff
@@ -0,0 +1,85 @@
+package dev.ebullient.convert.tools.pf2e;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import dev.ebullient.convert.tools.JsonNodeReader;
+import dev.ebullient.convert.tools.Tags;
+import dev.ebullient.convert.tools.pf2e.qute.QuteCreature;
+import dev.ebullient.convert.tools.pf2e.qute.QuteDataDefenses;
+
+public class Json2QuteCreature extends Json2QuteBase {
+
+    public Json2QuteCreature(Pf2eIndex index, JsonNode rootNode) {
+        super(index, Pf2eIndexType.creature, rootNode);
+    }
+
+    @Override
+    protected QuteCreature buildQuteResource() {
+        List<String> text = new ArrayList<>();
+        Tags tags = new Tags(sources);
+
+        appendToText(text, SourceField.entries.getFrom(rootNode), "##");
+
+        Collection<String> traits = collectTraitsFrom(rootNode, tags);
+        if (Pf2eCreature.alignment.existsIn(rootNode)) {
+            traits.addAll(toAlignments(rootNode, Pf2eCreature.alignment));
+        }
+        Optional<Integer> level = Pf2eCreature.level.getIntFrom(rootNode);
+
+        return new QuteCreature(sources, text, tags,
+                traits,
+                Field.alias.replaceTextFromList(rootNode, this),
+                Pf2eCreature.description.replaceTextFrom(rootNode, this),
+                level.orElse(null),
+                getPerception(),
+                buildDefenses());
+    }
+
+    // "perception": {
+    //     "std": 6
+    // }
+    private Integer getPerception() {
+        JsonNode perceptionNode = Pf2eCreature.perception.getFrom(rootNode);
+        if (perceptionNode == null) {
+            return null;
+        }
+        return perceptionNode.get("std").asInt();
```

Comment:
Just checking - is this a preference over adding a helper function in Pf2eTypeReader? Asking because the structure is this:
```
creature: {
  perception: {
    std: 10
  }
}
```
so `std` isn't actually a direct child of `creature`. This seems to be similar to what other methods in Pf2eTypeReader do for this. I could also add something like a `Pf2eNumberStat` which has a method to return the int value for the `std` key.

Reply:
Went ahead and just added it to `Pf2eCreature` as you've suggested, since I see that's what's been done for `Pf2eDataDefenses` - mostly because I didn't want to introduce another different way of doing it without converting the existing usages over, and that's more than I want to bite off right now :)